src/vs/workbench/contrib/debug/browser/rawDebugSession.ts

```diff
@@ -326,29 +328,43 @@ export class RawDebugSession implements IDisposable {
 	}
 
 	async next(args: DebugProtocol.NextArguments): Promise<DebugProtocol.NextResponse | undefined> {
+		this.stoppedSinceLastStep = false;
 		const response = await this.send('next', args);
-		this.fireSimulatedContinuedEvent(args.threadId);
+		if (!this.stoppedSinceLastStep) {
```

Comment:
This has a double `if`. Probably a typo.

Reply:
Too much cmd+D here 👍